openslides_votecollector/views.py

```diff
@@ -471,7 +472,7 @@ def post(self, request, poll_id, keypad_id):
         # Update votecollector.
         vc.votes_received = request.POST.get('votes', 0)
         vc.voting_duration = request.POST.get('elapsed', 0)
-        vc.save()
+        vc.save(skip_autoupdate=True)
```

Comment:
Proposal: Enable autoupdate and use `if collection_element.collection_string == '...vc...USE CORRET VALUE HERE`: return []` in projector.py as we do it for some other models. So the function does not have to run through the default else clause.